configurations/labwc/rc.xml

```diff
@@ -151,7 +154,7 @@
 
   <snapping>
     <!-- Set range to 0 to disable window snapping completely -->
-    <range>1</range>
+    <range>10</range>
```

Comment:
While my other two comments were rather questions, I think this one shouldn't be changed. See https://github.com/labwc/labwc/issues/2687#issuecomment-2821690819

Reply:
As far as I remember, only a value greater than 1 for this and `no` for `<xwaylandPersistence>` caused problems for me. The second problem should have been fixed now.